doc_src/en/HowTo_RestoreYourData.xml

```diff
@@ -139,7 +139,51 @@
 	</note>
   </section>
 
+  <section id="how.to.restore.your.data.project.locked">
+	<title id="how.to.restore.your.data.project.locked.title">Your project is
+	locked?</title>
 
+	<para>In the rare case where your computer freezes and OmegaT does not have
+	time to close properly, the <filename>omegat.project</filename> file can be
+	locked by the operating system and keep you from reopening after you reboot
+	your system.</para>
+
+	<para>Proceed as follows:</para>
+
+	<orderedlist>
+      <listitem>
+		<para>Identify the project folder that OmegaT refuses to open and open
+		it in your file manager. The folder contents should be similar to a
+		standard <link linkend="chapter.project.folder">project</link>
+		folder.</para>
+      </listitem>
+
+	  <listitem>
+		<para>In the folder, you should find a
+		<filename>omegat.project</filename> file as well as a
+		<filename>omegat.project.bak</filename> file. Make sure that the files
```

Comment:
https://github.com/omegat-org/omegat/commit/125ed163d9c20cf25de58bdd95c004b5b1741a59

Most recent OmegaT nightly does not produce `omegat.project.bak` but produce `omegat.project.YYYYMMDD.bak` and reuse it.

A change is proposed in July, 2021, and updated in Sep, 2022 then merged into master in Jan. 2023 at #415

Reply:
Please create a new issue on GH for documentation of the code modifications.

Reply:
I just created a new project and the backup file is not created. Can you check on your side?